CMakeLists.txt

```diff
@@ -56,7 +56,9 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 endif()
 
 if (GccSpecificFlags)
-  SET(CMAKE_CXX_FLAGS "-Wall -Wextra -pipe -O3 -g -ffast-math -flto -march=native -std=c++14 -DNDEBUG")
+  SET(GCC_COMPILE_FLAGS "-Wall -Wextra -pipe -O3 -g -ffast-math -flto -march=native -std=c++14 -DNDEBUG")
+  SET(GCC_DISABLED_WARNING_COMPILE_FLAGS "-Wno-ignored-attributes -Wno-maybe-uninitialized")
```

Comment:
Does the ignored attributes warning still happen? IIRC we pushed an update to cl2.hpp that fixes it?

Reply:
Yep, #798 fixed a couple of warnings, but the hardest to fix is ignored-attributes warning still left here: https://github.com/KhronosGroup/OpenCL-CLHPP/issues/37

Reply:
In fact I'm surprised you never saw these warnings?